temba/externals/views.py

```diff
@@ -4,11 +4,15 @@
 from django.utils.html import mark_safe
 from django.utils.translation import ugettext_lazy as _
 from django.urls import reverse
+from rest_framework.decorators import action
```

Comment:
`import action` is not being used, to make the code cleaner, unused imports are removed.

Reply:
Fixed